src/ir/context.rs

```diff
@@ -273,7 +283,6 @@ impl<'ctx> BindgenContext<'ctx> {
             currently_parsed_types: vec![],
             parsed_macros: Default::default(),
             replacements: Default::default(),
-            effective_target: effective_target.unwrap(),
```

Comment:
I think it makes sense to keep this around. As we improve cross compilation support, I imagine we will need this information.